README.md

```diff
@@ -123,6 +123,24 @@ val path = "my-file.json"
 val device: Device? = gson.fromResources(path, Device::class.java)
 ```
 
+### <a name="#common-usage"></a>Common Utility
+Please check the following sample code to use the common utility function (s).
```

Comment:
Change example to using extension classes.

Reply:
updated